docs/2_1_patching.md

```diff
@@ -7,23 +7,24 @@ The following pages will guide you through using ReVanced Manager to patch appli
 1. Navigate to the **Patcher** tab from the bottom navigation bar
 2. Tap on the **Select an application** card
 3. Choose an application to patch[^1]
-   > **Note**: The application version suggested by ReVanced Manager is visible in each application's card. This is important in step 5. below, as most patches are compatible with this version.
+   > **Note**: The application version suggested by ReVanced Manager is visible in each application's card. This is important for step 4. below, as most patches are compatible with this version.
 4. Tap on the **Select patches** card and select the patches you want to apply[^2]
    > **Warning**: If you see a warning icon next to a patch, it means that the patch is not compatible with the application version you selected in step 3. above. In this case, you should either go back and select a different application version or deselect the patch.
 5. Tap on the **Done** button to confirm your selection
 6. Tap on the **Patch** button to start patching
-   > **Warning**: The patching process can take anywhere from 2 to 5 minutes depending on your device. Refrain from exiting ReVanced Manager while the process is still running, otherwise the patching time will increase significantly.
+   > **Warning**: The patching process can take anywhere from 2 to 20 minutes depending on your device. Refrain from exiting the ReVanced Manager while the process is still running, otherwise the patching time will increase significantly.
```

Comment:
The average is 5
```suggestion
   > **Warning**: The patching process can take anywhere from 2 to 5 minutes depending on your device. Refrain from exiting the ReVanced Manager while the process is still running, otherwise the patching time will increase significantly.
```

Reply:
The wording "can take anywhere from x to y minutes.." implies that y is the maximum. How about "...usually takes 2 to 5 minutes, but varies depending on your device."

Reply:
Please check the `dev` branch. This is outdated. You generally don't work off a `main` branch.